frontend/app/routes/_public+/intake+/$id+/demographics-part1.tsx

```diff
@@ -49,6 +50,23 @@ export default function DemographicsPart1() {
             />
           )}
         </div>
+
+        <div className="my-6">
+          {disabilityTypes.length > 0 && (
+            <InputRadios
+              id="disability-type"
+              name="disabilityType"
+              helpMessage={t('demographics-oral-health-questions:part1.question4-note')}
```

Comment:
Let's leverage the changes made by https://github.com/DTS-STN/canadian-dental-care-plan/pull/545 to make this the `helpMessagePrimary`.

Reply:
done